tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
   "compilerOptions": {
     "module": "commonjs",
-    "target": "es6",
+    "target": "es2016",
```

Comment:
just for confirmation, are these updates necessary for the new package?

Otherwise I think they shouldn't be in this change (but are generally appreciated).

Reply:
The linter was suggesting the upgrade with some API I started using  and I upgraded it without checking it, but it's not really needed. I will remove it